Test/smoke-opera.js

```diff
@@ -0,0 +1 @@
+require('./smoke-test')('opera');
```

Comment:
There are the old test files when JavaScript was used, please let's use only Python.

Reply:
Ok, deleted.